Makefile

```diff
@@ -50,3 +56,11 @@ docker-serve:
 test-examples:
 	scripts/test_examples.sh install
 	scripts/test_examples.sh run
+
+link-checker-setup:
+	docker pull wjdp/htmltest
+
+run-link-checker:
+	docker run --mount type=bind,source=$(CURDIR),target=/test --rm wjdp/htmltest htmltest
+
+check-internal-links: build-linkcheck link-checker-setup run-link-checker
```

Comment:
This is the main target, that users need to run. Is my understanding correct? If so, add a comment?
1. Run hugo with special configuration to generate the site (HTML)
1. Pull the latest htmltest image from wjdp/htmltest
1. Run the wjdp/htmltest image in docker to evaluate the site